Makefile

```diff
@@ -43,7 +43,7 @@ all: binary docs
 # Then do the build and the output (skopeo) should appear in current dir
 binary: cmd/skopeo
 	docker build ${DOCKER_BUILD_ARGS} -f Dockerfile.build -t skopeobuildimage .
-	docker run --rm -v ${PWD}:/src/github.com/projectatomic/skopeo \
+	docker run --rm -v $$(pwd):/src/github.com/projectatomic/skopeo:Z \
```

Comment:
I think it would be better to disable selinux for this container then relabeling the skopeo directory.

--security-opt label=disable